Docs/medium guides (#206)
fetchai/fetchd: docs/docs/how-to-use-browser-wallet.md

# How to Use the Browser Extension Wallet

This is how to use the Browser Extension Wallet:

<img src="../images/browser_wallet.png" alt="FetchHub" class="center" style="display: block; margin-left: auto; margin-right: auto; width: 400px">
    

> <img src="../images/fetchhub.png" alt="FetchHub" class="center" style="display: block; margin-left: auto; margin-right: auto; width: 400px">

Reviewer: ```suggestion
<img src="../images/fetchhub.png" alt="FetchHub" class="center" style="display: block; margin-left: auto; margin-right: auto; width: 200px">
```

> 

Reviewer: ```suggestion
Open the extension.

<img src="../images/browser_wallet.png" alt="FetchHub" class="center" style="display: block; margin-left: auto; margin-right: auto; width: 300px">

To send some FET to another account, press Send:

img src="../images/fetchhub.png" alt="FetchHub" class="center" style="display: block; margin-left: auto; margin-right: auto; width: 300px">
```

# How to Use the Browser Extension Wallet

Open the Browser Extension Wallet.

   <img src="../images/browser_wallet.png" alt="FetchHub" class="center" style="display: block; margin-left: auto; margin-right: auto; width: 300px">

To send some FET to another account:

1. Press **Send**.

    <img src="../images/fetchhub.png" alt="FetchHub" class="center" style="display: block; margin-left: auto; margin-right: auto; width: 300px">

> 2. In the recipient field enter your native address.

Reviewer: ```suggestion
2. In the recipient field enter your native address. Leave Token type to FET.
```

> 3. Enter the amount you wish to send.

Reviewer: ```suggestion
3. Enter the amount you wish to send. Leave the "Memo" field empty.
```

4. Choose **Average** gas fees.

    You can set a custom gas fee, but that is not required in most cases.

    <img src="../images/fetchhub2.png" alt="FetchHub" class="center" style="display: block; margin-left: auto; margin-right: auto; width: 400px">

5. After entering all the details, the browser wallet will prompt you to review the operation and approve it if you are happy with it.

> That's it! Your transaction will be sent to the native address you input.

Reviewer: ```suggestion
Your transaction will now be sent to the native address you specified.
```

To see previous transactions you can also check out one of many different [block explorers](https://explore-fetchhub.fetch.ai/account/fetch1c2wlfqn6eqqknpwcr0na43m9k6hux94dp6fx4y) that have Fetch.ai as an option and input your mainnet address.

**Disclaimer - Although you can access test networks from the browser wallet, do not send TESTFET tokens to a Fetch.ai address on Cosmostation as TESTFET has no value and will not appear.**

## **FAQs**

1. **Can I send my ERC20 FET to this wallet?**

>     No. If you do, you will lose your tokens. The wallet can hold native FET tokens and not ERC20.

Reviewer: ```suggestion
    No. If you do, you will lose your tokens. The wallet can only hold native FET tokens and not ERC20 FET tokens.
```

2. **Can I use this wallet to send tokens from native to ERC20 without requiring a ledger?**

>     Yes. This was a highly requested feature from our community and we are happy to confirm you can deposit your native tokens from ledger or cosmostation on the     browser extension wallet and then send it via the [token bridge](https://token-bridge.fetch.ai/) (native to ERC20).

Reviewer: ```suggestion
    Yes. This was a highly requested feature from our community and we are happy to confirm that you can deposit your native tokens from the ledger or cosmostation on the browser extension wallet and then send it via the [token bridge](https://token-bridge.fetch.ai/) (native to ERC20).
```

3. **Will this wallet replace the token bridge?**

    No. The token bridge will continue to exist. The wallet will make the process easier to send native to ERC20 tokens without relying on a ledger.

4. **When can I download the Browser extension wallet?**

>       Browser extension wallet is available on [Chrome webstore](https://chrome.google.com/webstore/detail/fetchai-network-wallet/ellkdbaphhldpeajbepobaecooaoafpg?hl=en-GB).

Reviewer: ```suggestion
      Browser extension wallet is available now on [Chrome webstore](https://chrome.google.com/webstore/detail/fetchai-network-wallet/ellkdbaphhldpeajbepobaecooaoafpg?hl=en-GB).
```
